modules/acknowledgements/jsx/acknowledgementsIndex.js

```diff
@@ -213,7 +215,7 @@ class AcknowledgementsIndex extends Component {
         title='Add Acknowledgement'
         onClose={this.closeModalForm}
         show={this.state.showModal}
-        throwWarning={true}
+        throwWarning={(Object.keys(this.state.formData).length !== 0) && true}
```

Comment:
Why still keep the [&& true ] at this line?

Reply:
true haha i'll fix this!